Pandas_script.py

```diff
@@ -0,0 +1,267 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# # Pandas and plots
+# 
+# All data stored in `/data` folder
+
+# In[1]:
+
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
+import numpy as np
+from matplotlib.patches import ConnectionPatch
+
+
+# ## 'Real data'
+
+# ### Functions for reading .gff and .bed
+
+# For gff:
+
+# In[11]:
+
+
+def read_gff(filepath):
+    colnames = ['chromosome', 'source', 'type',
+                'start', 'end', 'score', 'strand',
+                'phase', 'attributes']
+    df = pd.read_csv(filepath, sep='\t', names=colnames, skiprows=1)
```

Comment:
Забыл оставить suggestion
```suggestion
    df = pd.read_csv(filepath, sep='\t', names=colnames, comment="#")
```